diff --git a/node_modules/stories-react/src/Components/Video/Video.component.tsx b/node_modules/stories-react/src/Components/Video/Video.component.tsx
index b82167b..b49361d 100644
--- a/node_modules/stories-react/src/Components/Video/Video.component.tsx
+++ b/node_modules/stories-react/src/Components/Video/Video.component.tsx
@@ -66,11 +66,8 @@ export function Video(props: IStoryComponentProps) {
       <div className={styles.soundIcon} onClick={() => setMute(!isMuted)}>
         <SoundIcon type={isMuted ? 'off' : 'on'} />
       </div>
-      {showLoader && (
-        <div className={styles.loaderWrapper}>
-          <div className={styles.loader} />
-        </div>
-      )}
+
+    
     </Fragment>
   );
 }
diff --git a/node_modules/stories-react/src/Components/Video/Video.styles.css b/node_modules/stories-react/src/Components/Video/Video.styles.css
index 2222024..7ad3d2f 100644
--- a/node_modules/stories-react/src/Components/Video/Video.styles.css
+++ b/node_modules/stories-react/src/Components/Video/Video.styles.css
@@ -15,6 +15,7 @@
   align-items: center;
   justify-content: center;
   align-content: center;
+  display: none;
 
 }
 
@@ -25,6 +26,7 @@
   width: 40px;
   height: 40px;
   animation: spin 1s linear infinite;
+  display: none;
 }
 
 .soundIcon {
@@ -37,7 +39,7 @@
   outline: none;
   border: none;
   cursor: pointer;
-
+  display: none;
 }
 
 @keyframes spin {
